Gruntfile.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function(grunt) {
```

Comment:
As a non-experienced JS dev I find it a bit strange that `./node_modules/.bin/grunt` runs a linter given that I know grunt can also be used for other things. I would find `./node_modules/.bin/grunt lint` much more intuitive, which may just be my lack of experience.